charts/prometheus-blackbox-exporter/templates/_helpers.tpl

```diff
@@ -228,15 +228,7 @@ containers:
   {{- end }}
   {{- end }}
   args:
-  {{- if .Values.config }}
-  {{- if .Values.configPath }}
-  - "--config.file={{ .Values.configPath }}"
-  {{- else }}
-  - "--config.file=/config/blackbox.yaml"
-  {{- end }}
-  {{- else }}
-  - "--config.file=/etc/blackbox_exporter/config.yml"
-  {{- end }}
+  - --config.file={{ .Values.configPath | default "/config/blackbox.yaml" }}
```

Comment:
As this is a major updat anyway we might think about to rename the var to `.Values.configFile` too.
Imho this fits better as the var currently contains a filename and not only the path.
Thoughts?